defi/src/utils/normalizeChain.ts

```diff
@@ -3750,10 +3750,11 @@ export const chainCoingeckoIds = {
   "Plume": {
     geckoId: "plume",
```

Comment:
please also remove the geckoId from the old chain since we cant have the same cg id in 2 places